src/core/asyncProcess.js

```diff
@@ -52,7 +52,7 @@ function asyncProcessFactory() {
          * @returns {boolean} - Returns true if the process can be started
          */
         start: function start(cb) {
```

Comment:
**quibble:** Converting the code to use `let` and `const` instead of `var` is good, but why not continue and also use the shorthand?

```suggestion
        start(cb) {
```